Mathlib/Data/Polynomial/Div.lean

```diff
@@ -164,6 +164,14 @@ theorem degree_modByMonic_lt [Nontrivial R] :
   termination_by degree_modByMonic_lt p q hq => p
 #align polynomial.degree_mod_by_monic_lt Polynomial.degree_modByMonic_lt
 
+theorem natDegree_modByMonic_lt [Nontrivial R] (p : R[X]) {q : R[X]} (hmq : Monic q) (hq : q ≠ 1) :
```

Comment:
Since you assume `q ≠ 1` you can remove `Nontrivial R`. You can try the [nontriviality](https://github.com/haruhisa-enomoto/mathlib4-all-tactics/blob/main/all-tactics.md#nontriviality) tactic to see if it is able to get `Nontrivial R` for you, or do it by hand.

Reply:
Done!